reporting-engine-worker/src/main/java/org/ihtsdo/termserver/scripting/reports/release/PackageComparisonReport.java

```diff
@@ -61,9 +85,39 @@ public Job getJob() {
 				.withExpectedDuration(40)
 				.build();
 	}
+
+	@Override
+	protected void loadProjectSnapshot(boolean fsnOnly) throws TermServerScriptException, InterruptedException, IOException {
+		//getArchiveManager(true).loadSnapshot(fsnOnly);
```

Comment:
These comments are not relevant to your use case.   Replace with a single comment explaining that you've overridden this function so as to avoid loading an archive into member.

Reply:
@pgwilliams old code